test/models/ability_test.rb

```diff
@@ -149,12 +128,12 @@ def teardown
       create_team_user user: u, team: t, role: 'admin'
       pm1 = create_project_media team: t
       pm2 = create_project_media team: t2
-      a1 = create_annotation annotated: pm1
-      a2 = create_annotation annotated: pm2
```

Comment:
This is why I suggest to replace `create_comment` with `create_tag`  https://github.com/meedan/check-api/pull/2219/files#r1963516751 by this way we can keep `create_annotation` as it.